### PR TITLE
Completed "import and create account" screen

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are adding the "import and create account" front-end screen.

image

### CODE REVIEW

File: web-frontend/src/layouts/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -9,7 +9,10 @@ interface LandingPageProps {
 
 export default function LandingPage(props: LandingPageProps) {
   return (
-    <div className="bg-brand bg-landing-page bg-no-repeat bg-cover bg-center min-h-screen">
+    <div
+      className={`bg-brand bg-landing-page bg-no-repeat bg-cover
```
Comment (pivilartisant) [inline web-frontend/src/layouts/LandingPage/LandingPage.tsx:13]: if you change the landing page here than some pages will need to be reformatted... maybe it's worth making a separate issue  for the update ?

Comment (maxime-carabina) [reply web-frontend/src/layouts/LandingPage/LandingPage.tsx:13]: Done

File: web-frontend/src/pages/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="w-fit h-fit max-w-lg">
+          <h1 className="mas-banner text-f-primary">
+            Welcome on
+            <br />
+            Massa<span className="text-brand">wallet</span>
+          </h1>
+          <div className="pt-6">
+            <Button posIcon={<FiArrowRight />}>Create an account</Button>
+          </div>
+          <div className="pt-3.5">
```
Comment (mazmassa) [inline web-frontend/src/pages/Welcome/Welcome.tsx:18]: 3.5!? 🤔

File: web-frontend/src/pages/Welcome/Welcome.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { FiArrowRight } from 'react-icons/fi';
+
+export default function Welcome() {
+  return (
+    <LandingPage>
+      <div className="flex flex-col justify-center items-center h-screen">
+        <div className="w-fit h-fit max-w-lg">
+          <h1 className="mas-banner text-f-primary">
+            Welcome on
```
Comment (mazmassa) [inline web-frontend/src/pages/Welcome/Welcome.tsx:11]: We should use il18n here but this will come after...

**Review by mazmassa — APPROVED** [review]
LGTM!

File: web-frontend/src/pages/Welcome/Welcome.tsx (hunk 2 above)
Comment (maxime-carabina) [reply web-frontend/src/pages/Welcome/Welcome.tsx:18]: image

@mazmassa Yes to have 14px space

**Review by Thykof — APPROVED** [review]

**Pull request merged**